### ISSUE TITLE
i18next returnObjects support

### ISSUE THREAD

**Issue by filipjakov** [lingualdev/i18n-check#26]
Hey guys,

Usually, the practice involves having statically analyzable keys, simplifying the parser's jobs. But we have codebase  instances where we have:

```tsx
// component.tsx
const resolutionLabels = t('resolutionLabels', { returnObjects: true });
...
const resolution = resolutionLabels[resolutionFetchedRemotly]
...

// en.json
...
"resolutionLabels": {
	"1m": "1 minute",
	"5m": "5 minutes",
	"15m": "15 minutes",
	"30m": "30 minutes",
	"1H": "1 hour",
	"4H": "4 hours",
	"1D": "1 day",
	"1W": "1 week",
	"1M": "1 month"
},
...
```

and `i18n-check` reports all the `resolutionLabels.{instance}` as false positives. 

Would it be possible to either:

- give i18n-check a set of keys to ignore
- detect `returnObjects` as a configuration option and somehow manage this problem

Or, as an alternative, do you have any recommendations on how to approach this problem?

**Comment by busypeoples** [comment]
@filipjakov let me check what we can do about this. If we could detect that this a `returnObjects` maybe we could mark them as in use internally. We could also create an option to ignore specific keys, which could make sense in general, as sometimes you might want to keep the keys for some reason - even if they are not used.

I will look into the issue and see what we can do to solve this issue. Technically we will probably add a configuration file, so it would be easier to define these options. Will update here, as soon as I have an idea.

**Comment by busypeoples** [comment]
Any entries with `returnObjects` set to true are now filtered out from the list of possible keys and are not shown as unused.

Will close this issue and create a new issue to solve the keys to ignore option. This will require a configuration to be defined via JSON or YAML first.
